data/kansei.json

```diff
@@ -4,5 +4,10 @@
     "fallen_front_limit": 445.0,
     "fallen_right_limit": 570.0,
     "fallen_left_limit": 400.0
+  },
+  "Imu" {
```

Comment:
Don't mix naming case in configuration file, use either classic `snake_case` or javascript's `camelCase`.